src/api/spaces/index.ts

```diff
@@ -19,4 +19,15 @@ export class SpacesAPI extends Request {
       params: params
     });
   }
+
+  /**
+   * Get spaces list.
+   */
+  public getSpacesList(this: Request<ISpace[]>) {
```

Comment:
I suggest simplifying the method name to `getSpaces` and adding the source key parameter:

```suggestion
  public getSpaces(this: Request<ISpace[]>, params?: IDataSourceParams) {
```

Reply:
what is the need of the sourcekey params ?